CHANGELOG.md

```diff
@@ -9,219 +9,221 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Fixed
 
-- Fix GRPC on HTTP2 POST method [PR #1419](https://github.com/3scale/apicast/pull/1419) [THREESCALE-9976](https://issues.redhat.com/browse/THREESCALE-9976)
+-   Fix GRPC on HTTP2 POST method [PR #1419](https://github.com/3scale/apicast/pull/1419) [THREESCALE-9976](https://issues.redhat.com/browse/THREESCALE-9976)
```

Comment:
why do you add a new space? Could be unintended change?

Reply:
My editor auto format file on save :sweat: . Fixed.

Since I've introduced a new change, I'll need another approval.